doc/hooks/meson.section.md

```diff
@@ -12,11 +12,11 @@ By default, `enableParallelBuilding` is enabled as Meson supports parallel build
 
 Controls the flags passed to `meson setup`.
 
-##### `mesonCheckFlags` {#mesoncheckflags}
```

Comment:
The headers where `###` and `####` for no apparent reason. I fixed them to be consistent.

Reply:
Do this in another pull request. It's better not to mix things.